### PR TITLE
LVM integration test

### PR DESCRIPTION
**Pull request by johnsonw**
Signed-off-by: Will Johnson <william.c.johnson@intel.com>

### CODE REVIEW

File: IML.IntegrationTest/IntegrationTest.fs [hunk 1]
```diff
@@ -23,12 +23,12 @@ let settle() = cmd "udevadm settle" >> ignoreCmd
 let rbSettle() = rbCmd "udevadm settle"
 let sleep seconds = cmd (sprintf "sleep %d" seconds)
 let scannerInfo =
-    (fun _ ->
-    pipeToShellCmd "echo '\"Stream\"'"
+    (fun _ ->
```
Comment (tanabarr) [inline IML.IntegrationTest/IntegrationTest.fs]: nit: trailing whitespace is added in this file

File: IML.IntegrationTest/IntegrationTest.fs [hunk 2]
```diff
@@ -114,20 +114,34 @@ testAsync "add multipath device" <| fun () ->
...
-        do! MdRaid.MdRaidCommand.createRaidAndRollback "/dev/sd[d-e]" "/dev/md0"
+        do! MdRaid.MdRaidCommand.createRaidAndRollback "/dev/sd[d-e]" "/dev/md0" 
                 [ "/dev/sdd1"; "/dev/sde1" ]
         do! settle()
         do! Filesystem.mkfs "ext4" "/dev/md0"
         return! scannerInfo
     }
     |> startCommand "add mdraid"
     |> Promise.map serializeDecodedAndMatch
+testAsync "add logical volume" <| fun () ->
```
Comment (tanabarr) [inline IML.IntegrationTest/IntegrationTest.fs]: nit: no new-lines between tests

File: IML.IntegrationTest/MdRaid.fs [hunk 3]
```diff
@@ -45,7 +45,7 @@ type private Create =
 
 module private Create =
     let private levelOption = sprintf "%s --level=%s"
-
+
```
Comment (tanabarr) [inline IML.IntegrationTest/MdRaid.fs]: nit: trailing whitespace added to this file

**Comment by tanabarr** [comment]
lots of trailing whitespace added in this patch, I'm assuming that's not intentional so should probably be removed

File: IML.IntegrationTest/IntegrationTest.fs (hunk 1 above)
Comment (jgrund) [reply IML.IntegrationTest/IntegrationTest.fs]: Looks like:

https://github.com/dungpa/fantomas/issues/250

**Pull request merged**